samples/wasi-threads/wasm-apps/no_pthread.c

```diff
@@ -35,12 +38,12 @@ wasi_thread_start(int thread_id, int *start_arg)
 int
 main(int argc, char **argv)
 {
-    shared_t data = { 0, 52 };
-    __wasi_errno_t err;
+    shared_t data = { 0, 52, -1 };
```

Comment:
```suggestion
    shared_t data = { .th_ready = 0, .value = 52, .thread_id = -1 };
```

Reply:
iirc, we intentionally avoid designated initializers for some reasons. (old compilers?)

Reply:
yes, I think this is only available in >= c99